### PR TITLE
User management

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ODAA-75

### CODE REVIEW

File: src/Security/DataFlowVoter.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            // the user must be logged in; if not, deny access
+            return false;
+        }
+
+        // ROLE_SUPER_ADMIN can do anything! The power!
+        if ($this->security->isGranted('ROLE_SUPER_ADMIN')) {
+            return true;
+        }
+
+        // you know $subject is a Post object, thanks to supports
```
Comment (lats1) [inline src/Security/DataFlowVoter.php]: Sneaky comment from the symfony docs should probably be removed.

File: templates/user/_delete_form.html.twig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<form method="post" action="{{ path('user_delete', {'id': user.id}) }}" onsubmit="return confirm('Are you sure you want to delete this item?');">
+    <input type="hidden" name="_method" value="DELETE">
+    <input type="hidden" name="_token" value="{{ csrf_token('delete' ~ user.id) }}">
+    <button class="btn">Delete</button>
+</form>
```
Comment (lats1) [inline templates/user/_delete_form.html.twig]: Missing translations

File: templates/user/edit.html.twig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{% extends 'layout.html.twig' %}
+
+{% block title 'Edit User'|trans %}
+
+{% block header 'Edit User'|trans %}
+
+{% block content %}
+    {{ include('user/_form.html.twig', {'button_label': 'Update'}) }}
+
+    <a href="{{ path('user_index') }}">back to list</a>
```
Comment (lats1) [inline templates/user/edit.html.twig]: Missing translations

File: templates/user/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+{% block content %}
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Email</th>
+                <th>Enabled</th>
+                <th>LastLogin</th>
+                <th>Roles</th>
+                <th>actions</th>
```
Comment (lats1) [inline templates/user/index.html.twig]: Missing translations

File: templates/user/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            </tr>
+        </thead>
+        <tbody>
+        {% for user in users %}
+            <tr>
+                <td>{{ user.email }}</td>
+                <td>{{ user.enabled ? 'Yes' : 'No' }}</td>
```
Comment (lats1) [inline templates/user/index.html.twig]: Missing translations

File: templates/user/index.html.twig [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+                <td>{{ user.lastLogin ? user.lastLogin|date('Y-m-d H:i:s') : '' }}</td>
+                <td>{{ user.roles ? user.roles|join(', ') : '' }}</td>
+                <td>
+                    <a href="{{ path('user_show', {'id': user.id}) }}">show</a>
+                    <a href="{{ path('user_edit', {'id': user.id}) }}">edit</a>
```
Comment (lats1) [inline templates/user/index.html.twig]: Missing translations

File: templates/user/index.html.twig [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+                </td>
+            </tr>
+        {% else %}
+            <tr>
+                <td colspan="13">no records found</td>
```
Comment (lats1) [inline templates/user/index.html.twig]: Missing translations

**Review by lats1 — CHANGES_REQUESTED** [review]

File: src/Security/DataFlowVoter.php (hunk 1 above)
Comment (rimi-itk) [reply src/Security/DataFlowVoter.php]: OMG! Busted. Again …

**Review by lats1 — APPROVED** [review]

**Pull request merged**